src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs

```diff
@@ -481,7 +481,7 @@ public Color BorderlineColor
         /// If 1.0f, borderline draw outside of View.
```

Comment:
479: Recommended range: [-1.0f  to 1.0f].
480, 481, 482: borderline draw -> borderline is drawn  OR  draw the borderline
482:  drop both "at"

Reply:
Fix done.